src/components/flashcard/Flashcard.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import "./Flashcard.css";
+
+const Flashcard = ({ question, answer }) => {
```

Comment:
We will most likely pass an entire flashcard object to this component instead of just the question and answer. You can change the prop to `flashcard` or `currentCard` or something else that makes sense to you here.

Reply:
Thanks, Nikki! At the moment of creating this component, it was unclear which attributes we would use as props. But I absolutely agree that in this case we should pass the entire flashcard object!